app/helpers/eitc_zendesk_instance.rb

```diff
@@ -58,6 +60,8 @@ class EitcZendeskInstance
     "refundday-h" => ONLINE_INTAKE_HU_FL,
     hispanicunity: ONLINE_INTAKE_HU_FL,
     uwfm: ONLINE_INTAKE_UW_FRESNO_MADERA,
+    uwcrpa: ONLINE_INTAKE_UWCRPA,
+    uwgc: ONLINE_INTAKE_GREENVILLE,
```

Comment:
Let's use the following variable names for consistency:
```
ONLINE_INTAKE_UW_CRPA,
ONLINE_INTAKE_UW_GREENVILLE,
``

Reply:
Will do!